exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -862,16 +1097,18 @@ func (ku *Kucoin) CancelAllOrders(ctx context.Context, orderCancellation *order.
 
 // GetOrderInfo returns order information based on order ID
 func (ku *Kucoin) GetOrderInfo(ctx context.Context, orderID string, pair currency.Pair, assetType asset.Item) (*order.Detail, error) {
-	if err := ku.CurrencyPairs.IsAssetEnabled(assetType); err != nil {
+	err := ku.CurrencyPairs.IsAssetEnabled(assetType)
+	if err != nil {
 		return nil, err
 	}
-	pair, err := ku.FormatExchangeCurrency(pair, assetType)
+	pair, err = ku.FormatExchangeCurrency(pair, assetType)
 	if err != nil {
 		return nil, err
 	}
 	switch assetType {
 	case asset.Futures:
-		orderDetail, err := ku.GetFuturesOrderDetails(ctx, orderID)
+		var orderDetail *FuturesOrder
```

Comment:
Please add test coverage to this futures case scenario

Reply:
We already have a test for that 
https://github.com/samuael/gocryptotrader/blob/0a453fb33b9f4a2582627205dc0cbc911c4baf60/exchanges/kucoin/kucoin_test.go#L2207
The map _assertToTradablePairMap_ has futures asset type and corresponding currency pair.

Reply:
I'll double check. I was going off of code coverage from tests with API keys and was not being hit

Reply:
I see now. Please change from `require` to `assert` in loops like that. Was preventing the other test scenarios from triggering

Reply:
Wouldn't it hide the error messages as assert is not a blocking call?

Reply:
`require` acts like a `t.Fatal`, stopping the test on the first error. `assert` acts like `t.Error`, it raises the error but continues with the rest of the test. Something like `TestGetOrderInfo` in a loop is going to fail on a scenario because its not setup to get an order for different IDs across each asset, so for it to stop testing on the first fail means it never tests the other scenarios